tests/test_core.py

```diff
@@ -1690,7 +1392,6 @@ def test_inherit(self):
 
     self.do_run_from_file(src, output)
 
-  @no_wasm_backend
```

Comment:
Some of them do. Some of them *did* and have regressed in the last week (or were spuriously passing in the first place?).